src/main/java/com/asu/cse545/group12/validator/TransactionInputValidator.java

```diff
@@ -0,0 +1,138 @@
+package com.asu.cse545.group12.validator;
+
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
+
+import com.asu.cse545.group12.domain.Form;
+import com.asu.cse545.group12.domain.TransactionForm;
+import com.asu.cse545.group12.domain.Users;
+
+import org.springframework.validation.ValidationUtils;
+
+
+public class TransactionInputValidator implements Validator{
+
+	public boolean supports(Class aClass) {
+		//just validate credit inputs
+		return Form.class.isAssignableFrom(aClass);
+	}
+
+	public void validate(Object obj, Errors errors) {
+		TransactionForm form = (TransactionForm) obj;
+
+		if(form.getTransactionType().equals("credit"))
+		{
+			try
+			{
+				Integer toAccountNumber;
+				if(form.getToAccount().equals(""))
```

Comment:
This may fail incase if string is passed.. as we dont have bean specifying any annotations to validate..do you see the problem.